hi-ml-histopathology/src/histopathology/datamodules/panda_module.py

```diff
@@ -32,3 +34,63 @@ def get_splits(self) -> Tuple[PandaTilesDataset, PandaTilesDataset, PandaTilesDa
         return (PandaTilesDataset(self.root_path, dataset_df=splits.train),
                 PandaTilesDataset(self.root_path, dataset_df=splits.val),
                 PandaTilesDataset(self.root_path, dataset_df=splits.test))
+
+
+class SubPandaTilesDataModule(TilesDataModule):
```

Comment:
We might want to call this class differently because its peculiarity is more in how the splits are built (it could be a subset or the whole dataset) and the fact it's not compatible with cross-validation.

On the longer-term cross-validation is something we want to build. Maybe worth to reason a second if it's easier to build that compatibility after or we should do it from the start.

Reply:
I see your point for cross validation. I was just trying to match the subsets used in the MONAI pipeline. That's why I hard fixed the train/validation sets based on the csv files I create from the MONAI json file.

I should go through the MONAI code once again but I'm not sure they're doing cross-validation.

It definitely needs to be renamed, any suggestions?